usvm-ts/src/test/kotlin/org/usvm/util/TSMethodTestRunner.kt

```diff
@@ -0,0 +1,178 @@
+package org.usvm.util
+
+import org.jacodb.ets.base.EtsType
+import org.jacodb.ets.dto.EtsFileDto
+import org.jacodb.ets.dto.convertToEtsFile
+import org.jacodb.ets.model.EtsFile
+import org.jacodb.ets.model.EtsMethod
+import org.usvm.PathSelectionStrategy
+import org.usvm.TSMachine
+import org.usvm.TSMethodCoverage
+import org.usvm.TSTest
+import org.usvm.UMachineOptions
+import org.usvm.test.util.TestRunner
+import org.usvm.test.util.checkers.ignoreNumberOfAnalysisResults
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.milliseconds
+
+open class TSMethodTestRunner : TestRunner<TSTest, MethodDescriptor, EtsType?, TSMethodCoverage>() {
+
+    protected val globalClassName = "_DEFAULT_ARK_CLASS"
+
+    protected inline fun <reified R> discoverProperties(
```

Comment:
Do we plan to somehow translate TS objects into Java objects?

Reply:
Yes, in the future they will be represented probably as a map from a property name to its value